app/views/provider_interface/account_creation_in_progress.html.erb

```diff
@@ -6,13 +6,24 @@
       Your account is not ready yet
     </h1>
     <p class="govuk-body">
-      You’re seeing this page because you’ve signed in
-      to the <%= t('service_name.apply') %> service but your account
-      is not ready to use yet.
+      You have signed in to DfE as
```

Comment:
Would adding this additional info still follow security good practice?
https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Authentication_Cheat_Sheet.md#authentication-and-error-messages

Reply:
It's good to check these things, but I think the inclusion of an email address on this page is ok, as they have already authenticated by the time they see this. It's more of an authorisation error (the authenticated user is not associated with any accounts). To perform an enumeration attack, they'd basically need log in as every user they are trying to assess.